phase1/src/initialization.rs

```diff
@@ -21,6 +21,7 @@ impl<'a, E: PairingEngine + Sync> Phase1<'a, E> {
         let one_g1 = &E::G1Affine::prime_subgroup_generator();
         let one_g2 = &E::G2Affine::prime_subgroup_generator();
 
+        #[cfg(not(feature = "wasm"))]
```

Comment:
I have a similar comment as in the other PR - I'd like to figure out a more friendly way so that we don't have to duplicate code.

Reply:
Take a look at https://github.com/AleoHQ/aleo-setup/pull/34

Reply:
cool, it looks really great, just it has nightly test failure. do you have any thought on that?

Reply:
Fixed :) I renamed something at the last minute.